### PR TITLE
Merge caption feature

### PR DESCRIPTION
**Pull request by ivee-tech**
TLD;DR Added caption #1

Fixes Issue #1 

## Done
- Added caption
- Linked Issue #1

## Todo
- [] Item 1
- [] Item 2

### CODE REVIEW

File: _slides/01-intro.md [hunk 1]
```diff
@@ -4,3 +4,5 @@ title: "Welcome to our slide deck!"
 ---
 
 Use the right arrow to begin!
+Caption added by ivee-tech
```
Comment (ivee-tech) [inline _slides/01-intro.md]: I like this! 👍

Comment (eyjsan) [inline _slides/01-intro.md]: I like this! 👍

Comment (ivee-tech) [inline _slides/01-intro.md]:
```suggestion
Caption added by ivee-tech today at 10 AM - additional suggestion
```

**Review by eyjsan — APPROVED** [review]

**Review by ivee-tech — COMMENTED** [review]
Approved

**Pull request merged**